pep-9999.rst

```diff
@@ -0,0 +1,350 @@
+PEP: 9999
```

Comment:
Please take PEP 719. Also, please add an entry to `.github/CODEOWNERS`.

Reply:
Looks like 719 is already taken, per #3184.

Reply:
Sorry for that! I looked only at active PRs. Let's use 720 instead.